src/codegen_call_io.cpp

```
@@ -190,10 +190,12 @@ llvm::Value *CodeGen::emitNetConnect(const CallExpr &e) {
     requireArgs(e, 2);
     llvm::Value *host = emitExpr(*e.args[0]);
     llvm::Value *port = emitExpr(*e.args[1]);
+    bool isTls = e.callee == "tls_connect";
     auto fn = mod_->getOrInsertFunction(
```

Clang (14.0.6)

[warning] variable name 'fn' is too short, expected at least 3 characters

(readability-identifier-length)